packages/documentation/src/content/docs/integration/deployment/services/backend-service.mdx

```diff
@@ -59,11 +61,11 @@ The `backend` service exposes a GraphQL [Backend Admin API](/apis/graphql/admin-
 
 ### Authentication and authorization
 
-To ensure the security and integrity of your Rafiki instance, all requests to the Backend Admin API must include a `tenant-id` header and be authenticated using HTTP signatures. This verifies that the request is from an authorized client. You are responsible for generating the HTTP signature and including the `tenant-Id` header in all requests. Rafiki then verifies the signature and enforces tenant-based access control. The [Rafiki Admin application](/admin/admin-user-guide/) automatically handles the creation of the HTTP signature and includes the necessary headers in the request.
+To ensure the security and integrity of your Rafiki instance, all requests to the Backend Admin API must include a `tenant-id` header and be authenticated using HMAC SHA-256 with the tenant's API secret. Rafiki validates the signature, derives the tenant context from the signed headers, and applies tenant-based access control. This HMAC-based request signing is distinct from HTTP Message Signatures used by Open Payments and from webhook verification (Rafiki-Signature). The [Rafiki Admin application](/admin/admin-user-guide/) automatically signs requests and includes the `tenant-id` header.
```

Comment:
I think we can leave out the "This HMAC-based request signing is distinct from HTTP Message Signatures used by Open Payments and from webhook verification (Rafiki-Signature)." part.

Also, we should specify that the signature is in the `signature` header of the HTTP request.
(Maybe also in `<TenantIdHmacNote/>`